cypress.config.js

```diff
@@ -1,8 +1,32 @@
 const { defineConfig } = require('cypress');
+const facker = require('faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/automation-practice-form',
+    viewportHeight: 1080,
+    viewportWidth: 1320,
     setupNodeEvents(on, config) {
-    }
-  }
+      on("task", {
+        generateUser() {
+          randomIndex = Math.floor(Math.random() * 2);
+          genders = ['Male', 'Female', 'Other'];
+          hobbies = []
+          return {
+            firstName: faker.name.firstName(),
+            lastName: faker.name.lastName(),
+            email: faker.internet.email(),
+            gender: genders[randomIndex],
+            phoneNumber: faker.phone.phoneNumber('##########'),
+            birthDate: facker.date.past(),
```

Comment:
Possible typo error in the package name 'facker'. It should be 'faker'.

Reply:
Done